config/purposes/scrna_core_library_prep.yml

```
@@ -110,3 +110,46 @@ LRC GEM-X 5p GE Pool Norm:
   :type: IlluminaHtp::InitialStockTubePurpose
   :presenter_class: Presenters::SimpleTubePresenter
   :default_printer_type: :tube
+LRC GEM-X 5p CITE SUP:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+LRC GEM-X 5p CITE SUP XP:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate:
+    false # Passed when submission is built.
+    # close off the submission when the plate is created
+  :state_changer_class: StateChangers::AutomaticPlateStateChanger
+  :work_completion_request_type: ['limber_scrna_core_cdna_prep_gem_x_5p_gems_input', 'limber_scrna_core_cdna_pcr_cite']
```

```suggestion
  :work_completion_request_type: ['limber_scrna_core_cdna_pcr_cite']
```

It shouldn't need to close off gems input requests, should it? Think they go down the left-hand (gene expression) branch and get closed off on the `LRC GEM-X 5p cDNA PCR XP` plate?